app/javascript/src/external_pledges.js

```diff
@@ -0,0 +1,10 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+
+jQuery(() =>
```

Comment:
No need to declare `jQuery`, we have it autoloaded thanks to some magic @rudietuesdays did a bit ago. You can slim this down to just L6-9.

(Getting this into regular javascript makes it so much easier to read and clean.)